src/commitizen/staging.js

```diff
@@ -1,20 +1,19 @@
-import git from 'gulp-git';
-import {isString} from '../common/util';
+import {exec} from 'child_process';
 
 export {isClean};
 
 /**
  * Asynchrounously determines if the staging area is clean
  */
 function isClean(repoPath, done) {
-  git.exec({cwd:repoPath, args: '--no-pager diff --cached --name-only', quiet: true}, function (err, stdout) {
-    let stagingIsClean;
-    if(stdout && isString(stdout) && stdout.trim().length>0)
-    {
-      stagingIsClean = false;
-    } else {
-      stagingIsClean = true;
+  exec('git diff --cached --name-only', {
```

Comment:
I think `--no-pager` isn't needed here, as the succeeding tests at [test/tests/staging](https://github.com/commitizen/cz-cli/blob/07c10335c22143f6421675a20e85a13e424aae23/test/tests/staging.js#L60) show. I guess the option was intended to guard against counting the length of the pager `(END)` string. According to my tests `git diff` output is not paged anyway in non-tty processes.